tests/view/crudapi/test_crud_api.py

```diff
@@ -30,3 +30,45 @@ def test_get_not_found_object(app):
         apiview = FakeAPIView()
         with pytest.raises(NotFound, message="Fake object not found"):
             apiview.get(0)
+
+
+def test_post_object(app):
+    """ Tests for CrudeAPIView post method. """
+    with app.test_request_context(
+            "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        payload = {"some_id": 6, "some_attr": "attr6"}
+        result = apiview.post(payload)
```

Comment:
@soin08 these 3 lines should be indented with -4 spaces. Same in the following 2 tests.